internal/services/v1/errors.go

```diff
@@ -337,6 +344,60 @@ func (err ErrInvalidCursor) GRPCStatus() *status.Status {
 	)
 }
 
+// ErrInvalidFilter indicates the specified relationship filter was invalid.
+type ErrInvalidFilter struct {
+	error
+}
+
+// GRPCStatus implements retrieving the gRPC status for the error.
+func (err ErrInvalidFilter) GRPCStatus() *status.Status {
+	// TODO(jschorr): Put a proper error reason in here.
+	return spiceerrors.WithCodeAndDetails(
+		err,
+		codes.InvalidArgument,
+		spiceerrors.ForReason(
+			v1.ErrorReason_ERROR_REASON_UNSPECIFIED,
```

Comment:
any reason to wait on defining a real reason here?

Reply:
It requires an API repo change and I didn't want to block this change, so I left it as a TODO